From 6c8599b682bbdbb32e3a5e4406be6403c16fb04c Mon Sep 17 00:00:00 2001
From: Tomi Korpipaa <tomi.korpipaa@qt.io>
Date: Tue, 7 Oct 2025 07:59:30 +0300
Subject: [PATCH 363/553] Fix documentation for QPrinter::PrinterResolution

Documentation omitted the fact that PrinterResolution
is not checked in QPdfPrintEngine code at all, and would
end up using the default value set in the code.

Pick-to: 6.8
Fixes: QTBUG-133410
Change-Id: I2424b46acf10db6da7d8e02b31132e6e0c273976
Reviewed-by: Volker Hilsheimer <volker.hilsheimer@qt.io>
Reviewed-by: Sami Varanka <sami.varanka@qt.io>
Reviewed-by: Titta Heikkala <titta.heikkala@qt.io>
(cherry picked from commit c88dfa3a75ab7bc8c749ae2217d02be7e50494ec)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/printsupport/kernel/qprinter.cpp | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/printsupport/kernel/qprinter.cpp b/src/printsupport/kernel/qprinter.cpp
index 6dc3c90460b..e17602e2882 100644
--- a/src/printsupport/kernel/qprinter.cpp
+++ b/src/printsupport/kernel/qprinter.cpp
@@ -335,11 +335,11 @@ public:
     lower quality output than HighResolution and should only be used
     for drafts.
 
-    \value PrinterResolution This value is deprecated. It is
-    equivalent to ScreenResolution on Unix and HighResolution on
-    Windows and Mac. Due to the difference between ScreenResolution
-    and HighResolution, use of this value may lead to non-portable
-    printer code.
+    \value PrinterResolution This value is deprecated. For printers,
+    it is equivalent to ScreenResolution on Unix and HighResolution
+    on Windows and macOS. For PDF printing, it is not supported and
+    may result in undefined behavior. Using this value can lead to
+    non-portable printer code.
 
     \value HighResolution On Windows, sets the printer resolution to that
     defined for the printer in use. For PDF printing, sets the
-- 
2.51.2

